src/containers/Playlists/PlaylistCard/index.js

```diff
@@ -47,13 +47,12 @@ class PlaylistCard extends React.Component {
 
   renderResources = () => {
     const { playlist } = this.props;
-    if (!playlist.resources || playlist.resources.length === 0) {
-      return (
-        <div className="alert alert-info m-3">No resources yet.</div>
-      );
+
+    if (!playlist.activities || playlist.activities.length === 0) {
+      return <div className="alert alert-info m-3">No resources yet.</div>;
```

Comment:
Remove this change

Reply:
playlist.resource is changes to activities now